src/Templates/src/templates/maui-mobile/MauiApp1/MauiApp1.csproj

```diff
@@ -17,15 +21,18 @@
 		<ApplicationVersion>1.0</ApplicationVersion>
 		<AndroidVersionCode>1</AndroidVersionCode>
 
-		<UseInterpreter Condition="'$(TargetFramework)' == 'net6.0-android'">True</UseInterpreter>
+		<!-- Required - Enable Single Project for WinUI -->
+		<EnablePreviewMsixTooling>true</EnablePreviewMsixTooling>
+		<WindowsPackageType>MSIX</WindowsPackageType>
```

Comment:
As far as I can tell, the EnablePreviewMsixTooling property needs to be in the csproj, it does not work in the Directory or the nuget.